torax/_src/sources/source_profiles.py

```diff
@@ -121,11 +121,14 @@ def merge(
     )
 
   def total_psi_sources(self, geo: geometry.Geometry) -> jax.Array:
-    total = self.bootstrap_current.j_bootstrap
-    total += sum(self.psi.values())
+    # All psi sources are assumed to be parallel to the magnetic field, ie
+    # self.psi.values() is <j.B> / B0
+    total_j_dot_B_over_B0 = self.bootstrap_current.j_bootstrap
+    total_j_dot_B_over_B0 += sum(self.psi.values())
+    total_j_dot_B = total_j_dot_B_over_B0 * geo.B_0
     mu0 = constants.CONSTANTS.mu0
-    prefactor = 8 * geo.vpr * jnp.pi**2 * geo.B_0 * mu0 * geo.Phi_b / geo.F**2
-    return -total * prefactor
+    prefactor = 8 * geo.vpr * jnp.pi**2 * mu0 * geo.Phi_b / geo.F**2
+    return -total_j_dot_B * prefactor
```

Comment:
This is mostly cosmetic stuff, just wanted to make it clear to anyone reading what was going on and for the equation to match the one in the paper.